install.rst

```diff
@@ -114,6 +114,23 @@ Setup the boot order: ::
 Finally, start the virtual machine.
 
 
+Install on VMWare
+-----------------
+
+`VMWare <https://www.vmware.com>`_ may encounter issues when importing raw disk images directly.
+To ensure a smooth import, convert the raw image to the VMWare native ``.vmdk`` format before proceeding.
```

Comment:
Maybe we should specify that the file should be extracted first, ie :

To ensure a smooth import, first extract the file, then convert the raw image to the VMWare native ``.vmdk`` format before proceeding.